scripts/read.py

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
Maybe you can run `black` on this file? (Or even all Python files, but here I noticed the weird newlines.)